app.js

```diff
@@ -22,5 +23,36 @@ function handleError(error) {
     console.warn(`Error displaying position, error: ${error.code}`)
 }
 
-navigator.geolocation.getCurrentPosition(printLocation, handleError);
+navigator.geolocation.getCurrentPosition(getLocation, handleError);
+
+// Getting the temperature from the Open Weather API using geographical coordinates.
+// Response format is Json by default.
+// https://openweathermap.org/current --free
+
+const KEY = "3137461397c70af0e13c77fda97afa11"
+
+function getTemp(latitude, longitude){
+    let api = `http://api.openweathermap.org/data/2.5/weather?lat=${latitude}&lon=${longitude}&appid=${KEY}&units=metric`
+    //the fetch() method instructs the web browsers to send a request to a URL.
+    //less than60 calls/min
+    fetch(api)
```

Comment:
:+1:

Reply:
In the constant meaning, maybe I'm wrong but urlApi is also similar to lat a long, it gets assign and then it is used as a constant.